src/tests/Microsoft.PowerFx.Core.Tests.Shared/DisplayNameTests.cs

```diff
@@ -787,5 +788,39 @@ public void RenameParameterTextFirst(string expr, string expected, bool find)
                 Assert.Equal(expected, renamer.ApplyRename(expr));
             }
         }
+
+        [Fact]
+        public void SummarizeDisplayNameTest()
+        {
+            var expression = "First(Summarize(DS, logicalField, logicalField2)).";
+            var record = RecordType.Empty().Add("logicalField", FormulaType.Number, "DisplayField").Add("logicalField2", FormulaType.String, "DisplayFiled2");
+
+            var symbol = new SymbolTable();
+            symbol.AddVariable("DS", record.ToTable());
+            symbol.AddFunction(new SummarizeFunction());
+
+            var config = new PowerFxConfig
+            {
+                SymbolTable = symbol
+            };
+
+            var engine = new Engine(config);
+
+            var logicalCheck = engine.Check(expression + "logicalField");
+            var displayCheck = engine.Check(expression + "DisplayField");
+
+            Assert.True(logicalCheck.IsSuccess);
+            Assert.True(displayCheck.IsSuccess);
+
+            // Check if intellisense suggests the display name.
+            var intellisenseResult = engine.Suggest(expression, RecordType.Empty(), expression.Length);
+
+            Assert.NotNull(intellisenseResult);
+            Assert.NotNull(intellisenseResult.Suggestions);
+
+            var suggestions = intellisenseResult.Suggestions.Select(sugg => sugg.DisplayText.Text);
+            Assert.Contains("DisplayField", suggestions);
```

Comment:
Testing if the display names will be suggested by the intellisense.

Reply:
nice!